.travis.yml

```diff
@@ -1,12 +1,37 @@
 language: node_js
 dist: trusty
-node_js:
-  - 6
-  - 7
+cache: yarn
 env:
+  global:
   - CXX=g++-4.8
-cache: yarn
 
-before_install:
-  - curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version 1.1.0
-  - export PATH="$HOME/.yarn/bin:$PATH"
+jobs:
+  include:
+    - env: nodejs=6
```

Comment:
Do we need this with the line below?

Reply:
This can help we understand which test we are running on the CI page, not affect the actual environment in our case at all.

Reply:
Makes sense, yes ^^

Reply:
It's shown here:
![image](https://user-images.githubusercontent.com/8719867/32122171-bf92e37a-bb5f-11e7-8bc4-4edb9956d470.png)